sources/core/Stride.Core/Native/NativeLibraryHelper.cs

```diff
@@ -12,19 +12,22 @@ namespace Stride.Core
     public static class NativeLibraryHelper
     {
         private const string UNIX_LIB_PREFIX = "lib";
-        private static readonly Dictionary<string, IntPtr> LoadedLibraries = new Dictionary<string, IntPtr>(StringComparer.OrdinalIgnoreCase);
-        private static readonly Dictionary<string, string> NativeDependencies = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+        private static readonly Dictionary<string, IntPtr> LoadedLibraries = new(StringComparer.OrdinalIgnoreCase);
+        private static readonly Dictionary<string, string> NativeDependencies = new(StringComparer.OrdinalIgnoreCase);
 
         /// <summary>
         /// Try to preload the library.
         /// This is useful when we want to have AnyCPU .NET and CPU-specific native code.
-        /// Only available on Windows for now.
         /// </summary>
         /// <param name="libraryName">Name of the library, without the extension.</param>
         /// <param name="owner">Type whose assembly location is related to the native library (we can't use GetCallingAssembly as it might be wrong due to optimizations).</param>
+        /// <param name="skipLibrary">If set to true, skips library loading<param>
         /// <exception cref="System.InvalidOperationException">Library could not be loaded.</exception>
-        public static void PreloadLibrary(string libraryName, Type owner)
+        public static void PreloadLibrary(string libraryName, Type owner, bool skipLibrary = false)
```

Comment:
What's the rationale here? If you don't want to load the library, just don't call the method from the caller site.

Reply:
done